spec/requests/api/v0/highlights_controller_spec.rb

```diff
@@ -5,6 +5,63 @@
 
   before { allow(Rails.application.config).to receive(:consider_all_requests_local) { false } }
 
+  describe 'GET /highlights' do
+    let!(:highlight1) { create(:highlight, user_uuid: user_uuid) }
+    let!(:highlight2) { create(:highlight, user_uuid: user_uuid) }
+    let!(:highlight3) { create(:highlight) }
+
+    let(:per_page) { 10 }
```

Comment:
should have some specs that don't set this or `page` and that will show you the kaboom